test/integer-types.js

```diff
@@ -523,11 +526,12 @@ describe("WebIDL unsigned long long type", () => {
     ], { enforceRange: true }, "with [EnforceRange]");
 });
 
-describe.skip("WebIDL unsigned long long test cases hard to pass within JavaScript", () => {
+describeBigInt("WebIDL unsigned long long test cases hard to pass within JavaScript", () => {
     // See the README for more details.
     const sut = conversions["unsigned long long"];
 
     generateTests(sut, [
-        [18446744073709552000, -18446744073709552000]
+        // -18446744073709551232 converted to 64 bits is 384
+        [18446744073709552000, 384]
```

Comment:
Let's add `9223372036854776000` as a test case also. (Should be the number itself.) The suggestion with stringification also applies here.